ui/src/CurrentUser.ts

```diff
@@ -99,7 +99,9 @@ export class CurrentUser {
     };
 
     public tryAuthenticate = async (): Promise<AxiosResponse<IUser>> => {
+        this.authenticating = true;
```

Comment:
I've removed this line, as I think we don't want to show the loading spinner, when we try to reconnect after there was a successful connection. I've set this.authenticating to true on variable initialization, so it should behave the same.